<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addFace</class>
 <widget class="QDialog" name="addFace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>269</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>378</width>
     <height>232</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineEdit_group_id">
           <property name="maximumSize">
            <size>
             <width>89</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_user_id">
           <property name="maximumSize">
            <size>
             <width>89</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_detect_quality">
           <property name="minimumSize">
            <size>
             <width>89</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>89</width>
             <height>25</height>
            </size>
           </property>
           <property name="currentText">
            <string>NORMAL</string>
           </property>
           <item>
            <property name="text">
             <string>LOW</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>NORMAL</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>HIGH</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_image">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>140</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>140</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>点击选择图片</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QTextEdit" name="textEdit_user_info">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_group_id">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>group_id</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_uer_id">
           <property name="text">
            <string>user_id</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_detet_quality">
           <property name="text">
            <string>detect_quality</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_user_info">
           <property name="text">
            <string>user_info</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox_ok_cancel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_ok_cancel</sender>
   <signal>accepted()</signal>
   <receiver>addFace</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_ok_cancel</sender>
   <signal>rejected()</signal>
   <receiver>addFace</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
